Library/Homebrew/commands.rb

```diff
@@ -143,4 +143,21 @@ def find_commands(path)
             .select(&:file?)
             .sort
   end
+
+  def rebuild_internal_commands_completion_list
+    cmds = internal_commands + internal_developer_commands + internal_commands_aliases
+
+    file = HOMEBREW_REPOSITORY/"completions/internal_commands_list.txt"
```

Comment:
Actually, as-is this should be a Pathname so you can skip all the `Pathname()` below and just run all these arguments on `file`.